mlflow/environment_variables.py

```diff
@@ -611,3 +611,8 @@ def get(self):
 _USE_MLFLOW_LANGCHAIN_TRACER_FOR_RAG_TRACING = _BooleanEnvironmentVariable(
     "USE_MLFLOW_LANGCHAIN_TRACER_FOR_RAG_TRACING", False
 )
+
+#: Specify the timeout in seconds for Databricks endpoint HTTP request retries.
+MLFLOW_DATABRICKS_ENDPOINT_HTTP_RETRY_TIMEOUT = _EnvironmentVariable(
+    "MLFLOW_DATABRICKS_ENDPOINT_HTTP_RETRY_TIMEOUT", int, 500
+)
```

Comment:
We already have `MLFLOW_HTTP_REQUEST_TIMEOUT`. Any reason we can't reuse it?

Reply:
Oh, we can't reuse it, the `MLFLOW_DATABRICKS_ENDPOINT_HTTP_RETRY_TIMEOUT` has a different default value.

Reply:
Databricks SDK `retry_timeout_seconds` default value is 300 seconds, so we'd better keep its original default value.